README.md

```diff
@@ -273,6 +290,32 @@ associative array in the first argument:
 </div>
 ```
 
+You can also pass parameters to actions:
+
+```twig
+<div {{ stimulus_action('controller', 'method', 'click', { 'count': 3 }) }}>Hello</div>
+
+<!-- would render -->
+<div data-action="click->controller#method" data-controller-count-param="3">Hello</div>
+```
+
+You can chain calls if you prefer as there's also a `stimulus_action` filter:
+
+```twig
+<div {{ stimulus_action('controller', 'method', 'click', { 'count': 3 })|stimulus_action('other-controller', 'test') }}>
+    Hello
+</div>
```

Comment:
Here, just like my comment above, let's remove the old documentation for the "old" way of doing multiple actions. So, an example similar to this would move UP to that section and replace the old stuff.